integration-tests/it-common/src/main/java/fr/pilato/elasticsearch/crawler/fs/test/integration/elasticsearch/FsCrawlerTestTikaConfigPathIT.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to David Pilato (the "Author") under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Author licenses this
+ * file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package fr.pilato.elasticsearch.crawler.fs.test.integration.elasticsearch;
+
+import fr.pilato.elasticsearch.crawler.fs.client.ESMatchQuery;
+import fr.pilato.elasticsearch.crawler.fs.client.ESSearchRequest;
+import fr.pilato.elasticsearch.crawler.fs.settings.Fs;
+import fr.pilato.elasticsearch.crawler.fs.test.integration.AbstractFsCrawlerITCase;
+import fr.pilato.elasticsearch.crawler.fs.tika.TikaInstance;
+import java.io.File;
+import java.io.InputStream;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+
+/**
+ * Test tika config path crawler setting
+ */
+public class FsCrawlerTestTikaConfigPathIT extends AbstractFsCrawlerITCase {
+
+  @Test
+  public void test_tika_config_path() throws Exception {
+    TikaInstance.reloadTika();

```

Comment:
Is that needed? I understand it is needed in the context of unit tests, but I'm unsure here. Could you confirm?

Reply:
In my testing I was finding that the parser would not be reassigned to by the second test run (as the parser is a static variable). This meant that the second test would essentially be using the settings of the first test, and would therefor fail.

I've removed the bad_path test, as now that a bad tika config path errors out, I think testing is harder and less productive than it was before. Please let me know if you think there should be more tests in this file - if more are required, I believe the reloadTika call might need to be added back in (unless I've missed something!)